domains/pallets/executive/src/benchmarking.rs

```diff
@@ -0,0 +1,33 @@
+//! Benchmarking for `pallet-executive`.
+
+use super::*;
+use frame_benchmarking::v2::*;
+use frame_system::RawOrigin;
+
+#[cfg(test)]
+use crate::Pallet as Executive;
+
+fn get_runtime_code() -> Vec<u8> {
+    include_bytes!(
+        "../../../../crates/pallet-domains/res/evm_domain_test_runtime.compact.compressed.wasm"
+    )
+    .to_vec()
```

Comment:
Wow, I just noticed we carry the domain runtime as WASM blob for some reason. We should have some kind of TODO to get rid of it. Especially for benchmarking purposes it is important to have it up to date I think.

Reply:
This is used only in testing
In pallet domains to do runtime upgrades checks and in integration tests for the same
and now in this for benchmarking

I believe this is the latest upto date one I believe

Reply:
I meant it would be great to pull up to date version right from the latest runtime automatically rather than committing it into the repository and updating from time to time.

Reply:
The benchmarking is for each extrinsic in a pallet not for the runtime, we just need a proper-format runtime, it can be just a template runtime instead of a domain runtime.

Reply:
@NingLin-P not only proper formatted runtime but runtime we push needs to be upgraded in the sense it should have a spec_version bumped etc.. 

Polkadot uses kitchen-sink runtime, a shell for this and maybe we can use it too or create something similar 
But fetching the latest runtime itself might not work I believe but would be nice to fetch the runtime from latest runtime though